opa/Dockerfile

```diff
@@ -58,13 +58,32 @@ WORKDIR /daemontools/admin/daemontools-${DAEMONTOOLS_VERSION}
 
 RUN package/install
 
-FROM stackable/image/vector
+FROM stackable/image/vector as opa-builder
```

Comment:
I think stackable-base is sufficient as a builder image at this point.

Reply:
You would need to add 
```
"stackable-base": "1.0.0",
```
to `conf.py` tested it locally and it was sufficient. you can also add stackable-base to opa-bundle-builder afaik, but @siegfriedweber might have a veto for that.

Reply:
And could you add a hadolint ignore to this line? Look at Kafka for an example.